### PR TITLE
@/Roboshield fix mobile UI bugs

### PR DESCRIPTION
**Pull request by m453h**
## Description

This PR fixes RoboShield Mobile UI issues, namely:

1.  Anchor tag text style causes overlapping of content in subsequent line(s)
2.  Improper (partially) rendered  Hamburger Menu Icon.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
New Project
Screenshot 2024-07-04 at 15 42 08 (2)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
Isn't it better to just increase the [`lineHeight`](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#try_it) @m453h

**Comment by m453h** [comment]
> Isn't it better to just increase the [`lineHeight`](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#try_it) @m453h

Yeah that would also work, I went with `inline-block` and the `margin` to create consistent spacing (LTRB) between the bounding box of the `<a>` text and any other content that would come after

With inline-block and margin it would look like this :

Screenshot 2024-07-04 at 17 19 55

With line height it would look like this :

Screenshot 2024-07-04 at 17 20 32

**Comment by kilemensi** [comment]
> I went with inline-block and the margin to create consistent spacing (LTRB) between the bounding box of the <a> text and any other content that ...

If it works for you and Kipruto all good... I thought we wanted the link to still "flow" as the rest of text rather than completely standout.

File: apps/roboshield/src/assets/icons/menu-icon.svg [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
-<path d="M4 24H28" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-<path d="M4 16H28" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-<path d="M4 8H28" stroke="black" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
-</svg>
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
```
Comment (kilemensi) [inline apps/roboshield/src/assets/icons/menu-icon.svg:1]: We should have this [icon](https://www.figma.com/design/cm8L9aMFztSHx91F8MWXQr/Editable-CFA-Site-2023?node-id=62-343&t=S24MrDfMPCeFLCFz-0) in various sizes in the CfA site Figma file... what's up with svgrepo.com?

**Review by kilemensi — APPROVED** [review]

File: apps/roboshield/src/assets/icons/menu-icon.svg (hunk 1 above)
Comment (m453h) [reply apps/roboshield/src/assets/icons/menu-icon.svg]: Got it 👍 , I've used the standard icon

**Pull request merged**